_includes/tutorials/splitting/confluent/markup/dev/ccloud-run-producer.adoc

```diff
@@ -0,0 +1,11 @@
+In a new terminal, run:
+
+```
+ccloud kafka topic produce acting-events --value-format avro --schema src/main/avro/acting_event.avsc
+```
+
+When the console producer starts, it will log some messages and hang, waiting for your input. Type in one line at a time and press enter to send it. Each line represents an event. To send all of the events below, paste the following into the prompt and press enter:
```

Comment:
Do we still call it a `console producer` if using the `ccloud` CLI?

Reply:
That's up to you. I see them as the same and we do "console producer" here: https://github.com/confluentinc/kafka-tutorials/blob/master/_includes/shared/markup/ccloud/ccloud-sr-produce.adoc.